source/common/runtime/runtime_impl.cc

```diff
@@ -212,13 +233,22 @@ uint64_t SnapshotImpl::getInteger(const std::string& key, uint64_t default_value
   }
 }
 
+bool SnapshotImpl::getBoolean(const std::string& key, bool& value) const {
+  auto entry = values_.find(key);
+  if (entry != values_.end() && entry->second.bool_value_) {
```

Comment:
nit: `entry->second.bool_value_.has_value()` (since I can't remember whether it means the value or presence without it)